### DISCUSSION TITLE
maya2024无法加载插件如何解决

### DISCUSSION THREAD

**Discussion by BruceStarkV233** [3DTech-Steven7/MayaToRizomuv#12] [Q&A]
loadPlugin "MayaToRizomUV.py";
pluginInfo -edit -autoload true "MayaToRizomUV.py";
// 错误: line 1: NameError: file C:/Users/Administrator/Documents/MayaToRizomUV/plug-ins/MayaToRizomUV.py line 43: name 'mtor' is not defined
// 警告: line 1: Failed to call script initialize function
// 错误: line 1:  (MayaToRizomUV)
// 错误: line 2: 对象或值无效:
// 错误: file: C:/Program Files/Autodesk/Maya2024/scripts/others/pluginWin.mel line 316: NameError: file C:/Users/Administrator/Documents/MayaToRizomUV/plug-ins/MayaToRizomUV.py line 43: name 'mtor' is not defined
// 警告: file: C:/Program Files/Autodesk/Maya2024/scripts/others/pluginWin.mel line 316: Failed to call script initialize function
// 错误: file: C:/Program Files/Autodesk/Maya2024/scripts/others/pluginWin.mel line 316:  (MayaToRizomUV)

**Comment by CG-HOU** [comment]
https://www.gfxcamp.com/rizomuv-bridge-for-maya/
